troposphere/static/js/components/projects/resources/link/details/sections/ExternalLinkActions.react.js

```diff
@@ -16,11 +16,11 @@ define(function (require) {
 
     onDelete: function () {
       var project = this.props.project,
-        link = this.props.link;
+        external_link = this.props.link;
 
       modals.ExternalLinkModals.destroy({
-        link: this.props.link,
-        project: this.props.project
+        project,
+        external_link
```

Comment:
ugh.. so [Line 35](https://github.com/iPlantCollaborativeOpenSource/troposphere/blob/ATMO-1302-fix-broken-delete/troposphere/static/js/components/projects/resources/link/details/sections/ExternalLinkActions.react.js#L35) would be the line to call handleReport, the onClick never gets called because of  [line 57](https://github.com/iPlantCollaborativeOpenSource/troposphere/blob/ATMO-1302-fix-broken-delete/troposphere/static/js/components/projects/resources/link/details/sections/ExternalLinkActions.react.js#L57).

Also, I'm 99% sure that `modals.ExternalLinkModals.report` doesn't exist.

This PR fixes a critical bug on beta.

I vote we fix the reporting on another PR once we understand what the intent even was.